buildpipeline/Core-Setup-Signing-Windows-x86-master.json

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
I see two substantial differences between x64/x86: x86 is missing the "sign assemblies" step on x64 line 154, and x86 does not build NuGet packages.  A) are these intentional?  B) Are they something we could consolidate and have one build definition with an architecture parameter?  C) Or does core-setup care enough about architecture to keep them separate for build history reasons even if we could consolidate?